src/addrman.cpp

```diff
@@ -233,10 +233,22 @@ void CAddrMan::Good_(const CService& addr, int64_t nTime)
     if (nUBucket == -1)
         return;
 
-    LogPrint(BCLog::ADDRMAN, "Moving %s to tried\n", addr.ToString());
+    // which tried bucket to move the entry to
+    int tried_bucket = info.GetTriedBucket(nKey);
+    int tried_bucket_pos = info.GetBucketPosition(nKey, false, tried_bucket);
+
+    // Will moving this address into tried evict another entry?
+    if (test_before_evict && (vvTried[tried_bucket][tried_bucket_pos] != -1)) {
+        LogPrint(BCLog::ADDRMAN, "addrman", "Collision inserting element into tried table, moving %s to m_tried_collisions=%d\n", addr.ToString(), m_tried_collisions.size());
```

Comment:
I'm assuming this was a rebase issue, but the "addrman" param is unnecessary here. It's actually not clear to me what this would do.

Reply:
I think it will log "addrman" and nothing more.

Reply:
Whoops, sorry @theuni.
I think it will log "Error not enough conversion specifiers while formatting log message: addrman"

Reply:
See #12622